homeassistant/components/plugwise/quality_scale.yaml

```diff
@@ -0,0 +1,65 @@
+rules:
+  ## Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup: done
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions: done
+  brands: done
+  ## Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions: done
+  reauthentication-flow:
+    status: exempt
+    comment: This integration does not have reauthentication (e.g. fixed authentication for Plugwise Smiles)
+  parallel-updates:
+    status: done
+    comment: Using coordinator
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters: done
+  ## Gold
+  entity-translations: done
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default: done
+  discovery: done
+  stale-devices: done
+  diagnostics: done
+  exception-translations: done
+  icon-translations: done
+  reconfiguration-flow:
+    status: exempt
+    comment: This integration does not have any reconfiguration steps
+  dynamic-devices: done
```

Comment:
binary_sensor.py has this logic, but button.py not?

Reply:
Sorry what do you mean here?
Reboot vs Restart? Remove the `_attr_translation_key`?

Reply:
No, this is the unload part, it's in all others but button

Reply:
Oh, I didn't even spot that one. I think we had that discussion before. Ideally we use the device class name for consistency.

But what I meant here was, binary sensors are added on the fly, but button are not, why is that?

Reply:
I kept the code in `button.py` simple because there is only one button defined at the moment, and I don't foresee any more buttons being added in the future.

Also this button is connected to the gateway-device, it will never change.
When this device changes a new Plugwise integration must be added because the ID = pw changes.

Now that I write this, this could be a reason for adding reconfiguring!?